libs/designer-ui/src/lib/pager/pager.less

```diff
@@ -1,5 +1,36 @@
 @import (reference) '../variables.less';
 
+.msla-pager-v2-pagenumbers {
+  display: flex;
+  flex-direction: row;
```

Comment:
NIT: row is the default value for `display: flex`, there is no need to add it exta